app/controllers/application_controller.rb

```diff
@@ -20,6 +20,6 @@ def set_locale
   end
 
   def default_url_options
-    {}
+    super.except(:locale)
```

Comment:
Do we care that this drops locales if one other than `:en` is used?

It seems like we could do something like:

```ruby
opts = super

return opts unless opts[:locale] == :en

opts.except(:locale)
```

Reply:
Hmm, I'm not sure it needs to be that specific. The UI for switching locales was removed from
the header, so I'm not sure how they would switch the locale. @mark-dce what do you think, considering the user base at Tufts?

Reply:
Let's go ahead and merge this and we can reopen if `:locale` is an issue.